### DISCUSSION TITLE
Range Bar Chart with Dates

### DISCUSSION THREAD

**Discussion by webdesign2be** [ant-design/ant-design#40812] [General]
Hi!

I would like to use the antd charts to create a range bar chart that uses "from -> to" dates to define the ranges.
Right now I have only seen a range bar chart that uses number values.

Does anyone have an idea about this topic?

Thanks in advance for any answer!

kind regards

**Comment by Yuiai01** [comment]
Hi, I don't know if you are talking about [Ant Design Charts](https://antv.vision/en), if so, you can refer to this demo https://charts.ant.design/en/examples/column/stacked#basic
I hope I can help you.

**Reply by webdesign2be** [reply]
Hi, thank you for your response.
Unfortunately, that is not what I want.
I use the Range Bar (https://charts.ant.design/en/examples/bar/range#basic) to show the Dates as ranges.

Right now I convert the Dates to timestamps, therefor the dates are numbers and the chart is only working with numbers.
But it would be nice if there would be an option to use dates as values too.

**Reply by Yuiai01** [reply]
> Hi, thank you for your response. Unfortunately, that is not what I want. I use the Range Bar (https://charts.ant.design/en/examples/bar/range#basic) to show the Dates as ranges.
> 
> Right now I convert the Dates to timestamps, therefor the dates are numbers and the chart is only working with numbers. But it would be nice if there would be an option to use dates as values too.

Well, I'm sorry I didn't help you. But I think [ant-design-charts](https://github.com/ant-design/ant-design-charts/issues/new/choose) is a better way to ask questions.

**Reply by solosignet** [reply]
Did you find an answer? Can't find your issue in the charts repo.
